docs/content/tutorials/custom-resource-type/index.md

```diff
@@ -0,0 +1,275 @@
+---
+type: docs
+title: "Tutorial: Add a custom resource type"
+linkTitle: "Add a custom resource type"
+description: "Learn how to define and deploy a resource type in your Radius application"
+weight: 120
+categories: ["Tutorial"]
+---
+
+## Overview
+
+Radius includes several built-in resource types which developers can use to build applications. These include core resource types such as Containers, Gateways, and Secrets. You can also create your own custom resource types. This tutorial guides you through creating a PostgreSQL resource and deploying the sample Todo List application with PostgreSQL.
+
+{{< image src="tutorial2.png" alt="Diagram of the Todo List with PostgreSQL" width=600px >}}
+
+## Prerequisites
+
+This tutorial assumes you have completed the [Create a new application]({{< ref "/tutorials/new-app" >}}) tutorial and have Radius installed and the demo application deployed. 
+
+Additionally, you will need a location to store your Recipe:
+
+  - **Terraform** configurations must be stored in a Git repository. Ideally for this tutorial the Git repository has anonymous access. If not, you will need to configure [Git authentication]({{< ref "guides/recipes/terraform/howto-private-registry" >}}).
+  
+  - **Bicep** templates must be stored in an OCI registry. As with Git, you must have anonymous access to the registry or configure [authentication]({{< ref "guides/recipes/howto-private-bicep-registry" >}}).
+
+Finally, [Node.js](https://nodejs.org/en/download) must be installed on the workstation to generate the Bicep extension to deploy the new resource type. 
+
+<!-- TODO: Add option for users to skip the need for an OCI registry or Git repository for their recipe by showing a `rad recipe register` command which includes a reference to an already deployed recipe in the Radius project.  -->
+
+## Step 1: Create a PostgreSQL resource type in Radius
+
+To create a PostgreSQL resource type in Radius, first create the resource type definition then add the resource type to Radius.
+
+1. Create a new file called `types.yaml` and add the following:
+
+   {{% rad file="snippets/types.yaml" lang=YAML embed=true %}}
+
+   The PostgreSQL resource type definition includes:
+
+    - **`name`**: The namespace of the resource type, as a convention `Radius.Resources` is recommended but any name in the form `PrimaryName.SecondaryName` can be used
+    - **`types`**: The resource type name
+    - **`capabilities`**: This specifies features of the resource type. The only available option is `SupportsRecipes` which indicates that the resource type can be deployed via a Recipe. In a future release of Radius, `SupportsRecipes` will be the default and this line item will not be required.
```

Comment:
> In a future release of Radius, `SupportsRecipes` will be the default and this line item will not be required.

What's the purpose of including this note? Will the future release also add other options beyond `SupportsRecipes`?